pkg/geo/geomfn/binary_predicates.go

```diff
@@ -135,6 +136,8 @@ func Equals(a geo.Geometry, b geo.Geometry) (bool, error) {
 	return geos.Equals(a.EWKB(), b.EWKB())
 }
 
+var preparedCache = &geosprepared.Cache{}
```

Comment:
please put a TODO here saying this cache is global and should probably be short lived. i think we should let it live on builtins instead - see other comment below.